htsohm/db/material.py

```diff
@@ -289,3 +292,27 @@ def calculate_retest_result(self, tolerance):
         )
 
         return not retest_failed
+
+    def artificial_void_fraction(self):
+#        htsohm_dir = os.path.dirname(os.path.dirname(htsohm.__file__))
+#        config_file = os.path.join(htsohm_dir, self.run_id, 'config.yaml')
+#        config = load_config_file(config_file)
+
+        return pseudo_void_fraction(
+                config, self.structure.number_density(),
+                self.structure.average_sigma())
```

Comment:
it's a global variable -- something to discuss? the global config makes things less portable.